tests/integration/test_integration.py

```diff
@@ -271,6 +271,51 @@ def test(
         assert actual == data["expected"]
 
 
+class TestInitSbom:
+    class DataFixture(TypedDict):
+        expected: dict
+
+    @pytest.fixture(
+        scope="class",
+        params=[
+            {
+                "expected": "init-sbom.initial_expected.json",
+            }
+        ],
+    )
+    def data(self, data_dir: Path, request: pytest.FixtureRequest) -> DataFixture:
+        expected_path = data_dir / request.param["expected"]
+
+        expected_json = load_sbom(expected_path)
+
+        return self.DataFixture(
+            expected=expected_json,
+        )
+
+    def test(
```

Comment:
please also add a test for no args given, i.e. only `init-sbom`

Reply:
done